packages/dds/tree/src/test/feature-libraries/treeCursorUtils.spec.ts

```diff
@@ -53,86 +52,70 @@ describe("treeCursorUtils", () => {
 		});
 
 		it("wrapped root", () => {
-			assert(
-				compareUpPaths(prefixPath({ indexOffset: 1 }, root), {
+			expectEqualPaths(prefixPath({ indexOffset: 1 }, root), {
```

Comment:
expectEqualPaths does the same thing, but will produce nicer errors when failing.